models/addRecipe.js

```diff
@@ -0,0 +1,59 @@
+module.exports = function(sequelize, Sequelize) {
+  var addRecipe = sequelize.define("addRecipe", {
+        name: {
+            type: Sequelize.STRING,
+            notEmpty: true
+        },
+        recipeID: {
+            type: Sequelize.STRING,
+            notEmpty: true,
+            unique: true
+        },
+        time: {
+            type: Sequelize.STRING,
+            notEmpty: true
+        },
+        feeds: {
+            type: Sequelize.INTEGER,
+            notEmpty: true
+        },
+        ingredients: {
+            type: Sequelize.INTEGER,
+            notEmpty: true
+        },
+        calories: {
+            type: Sequelize.INTEGER,
+            notEmpty: true
+        },
+        imageUrl: {
+            type: Sequelize.STRING,
+            notEmpty: true
+        },
+        recipeUrl: {
+            type: Sequelize.STRING,
+            notEmpty: true
+        },
+        nutritionEstimates: {
+            type: Sequelize.TEXT,
+            notEmpty: true
+        },
+        calendar: {
+            type: Sequelize.TEXT,
+            notEmpty: true
+        }
+  });
+
+  addRecipe.associate = function(models) {
```

Comment:
I'm not going to reject the pull request based on this but when creating models, think of a model car. It's an object that has features and does stuff. So how that would relate to this model is, instead of addRecipe being our model, Recipe would be our model. The Recipe could add recipes much like the car could drive or add a wheel. Other examples would be our car could be red or our recipe could be Mexican or Italian.

Reply:
thanks David. I have been thinking of models as tables. so i have a separate model for favourites and a separate model for the calendar. they both some columns that are the same and some that are different. is there a better approach this?

Reply:
Thats a good way of thinking of it. I'd agree that you'd have a favourites table and a calendar table - but for another project not this one lol.